### PR TITLE
Qunit

### PR DESCRIPTION
**Pull request by kyle-ilantzis**
Adds http://qunitjs.com/

$> gulp test

to see the sample test in nwjs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pl-sql-client",
-  "version": "0",
+  "version": "0.0.0",
```
Comment (pierluc-codes) [inline package.json:3]: :(

File: src/html/index.html [hunk 2]
```diff
@@ -76,12 +77,9 @@
 	<link href="app:/_/theme-superhero.css" type="text/css" data-theme="superhero"/>
 <head>
 <body>
-	<script type="text/javascript">
+	<!-- substitute:main -->
 
-		$.noConflict();
```
Comment (pierluc-codes) [inline src/html/index.html:81]: no more noConflict() ?

File: package.json (hunk 1 above)
Comment (kyle-ilantzis) [inline package.json:3]: we need a semantic version for npm to accept our packa…

…ge.json

File: src/html/index.html (hunk 2 above)
Comment (kyle-ilantzis) [inline src/html/index.html:81]: $ is for jQuery

$.noConflict() removes $ from the global namespace

http://api.jquery.com/jQuery.noConflict/

**Pull request merged**